File: Python_Bootcamp/Week13_PYTHON/Day5/CHALLENGES2/Ex1.py
x = range(1,6)
counter = 1
for x in range(1,6):
    if x %2 != 0:
        print(' ' * int((6-x)/2) + '*' * x + ' ' * int((6-x)/2))

for x in range(1,6):
    print(' ' * int((6-x)) + '*' * x)

for x in range(1,11):
    y = 11 - x
    if x < 6:
        print('*' * x + ' ' * int((6-x)/2))
    else:
        print(' ' * (x-y-counter) + '*' * y )
        counter += 1

File: Python_Bootcamp/Week8_ADVANCED_JS/Day5 - Copy/DAILYCHALLENGE_TRUEFALSE/Ex1.js
function allTruthy(...args) {
    counter = 0
    for (let arg of args) {
        if (!arg) {
            counter = counter + 1
            break
        }
        }
    if (counter == 0) {
        console.log("true");
    }
    else {
        console.log("false");
    }
}

allTruthy(false, true, true)